Week 1: R skeleton & first function with description

Lorry7 opened the pull request:
includes the R skeleton, including the description file and a function to retrieve rocket launch data from an API and transform it into a consice and usable data frame. Function description is included as well.

Lorry7 committed:
Week 1: R skeleton & first function with description

includes the R skeleton, including the description file and a function to retrieve rocket launch data from an API and transform it into a consice and usable data frame. Function description is included as well.

catherinegua commented on RocketLaunches2024/.Rhistory:
I'm not sure if the R History is needed here.

saministeru commented:
Hey! It looks really good. You have the basic structure: description, namespace, file with functions. You function has documentation. For the example, I would go more into depth, such as by showing the first three rows from the API vs. three rows from you data frame. You can add the path to a separate R file in which you do this. Also, remember to add the required packages in your documentation file (under "Imports"). Great job so far!

saministeru reviewed (commented):
Already submitted as a comment

Lorry7 committed:
New Package

Added the new package (RocketsNEW) and deleted the old one that caused errors. Incorporated the peer feedback.
